### PR TITLE
Improve fingertip calibration performance

### PR DESCRIPTION
**Pull request by ChingT**
Adopt CNN methods for hand and fingertip detector to improve the accuracy and robustness of the detection. 
GPUs will be utilized if they are available.

### CODE REVIEW

File: pupil_src/shared_modules/calibration_routines/fingertip_calibration/fingertip_calibration.py [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+from pyglui import ui
+from pyglui.cygl.utils import draw_points, draw_polyline, draw_progress, RGBA
+from pyglui.pyfontstash import fontstash
+from pyglui.ui import get_opensans_font_path
+from ..calibration_plugin_base import Calibration_Plugin
+from ..finish_calibration import finish_calibration
+import torch
+from .models.unet import UNet
+from .models.ssd_lite import build_ssd_lite
+
```
Comment (papr) [inline pupil_src/shared_modules/calibration_routines/fingertip_calibration/fingertip_calibration.py]: Please use the fully qualified import instead of relative imports as I mentioned above.

File: pupil_src/shared_modules/calibration_routines/fingertip_calibration/__init__.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Pupil - eye tracking platform
+Copyright (C) 2012-2018 Pupil Labs
+
+Distributed under the terms of the GNU
+Lesser General Public License (LGPL v3.0).
+See COPYING and COPYING.LESSER for license details.
+---------------------------------------------------------------------------~(*)
+'''
+
+from . fingertip_calibration import Fingertip_Calibration
```
[the remark this answers is not in the document]
Comment (papr) [reply pupil_src/shared_modules/calibration_routines/fingertip_calibration/__init__.py]:
```python
try:
    import torch
    from . fingertip_calibration import Fingertip_Calibration
except ImportError:
    from calibration_routines.fallback_fingertip_calibration import Fingertip_Calibration
```
`fallback_fingertip_calibration.py` would implement a dummy calibration plugin that shows an info text with a warning that torch is required for the actual plugin to work.
